libs/stack/stack-ui/src/components/Carousel/swiper/useCarouselSlide.tsx

```
@@ -31,28 +20,27 @@ export function useCarouselSlide(props: TCarouselSlideProps): TCarouselSlide {
   const a11yParams = typeof swiper?.params?.a11y === 'object' ? swiper.params.a11y : undefined
   const { itemRoleDescriptionMessage = 'slide', slideRole = 'group' } = a11yParams ?? {}
   const hasTitle = !isEmpty(title)
+  const { activeIndex, controller } = useCarousel() ?? {}
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)